pyproject.toml

```diff
@@ -14,6 +14,9 @@ fsspec = "^2024.6.1"
 gcsfs = "^2024.6.1"
 gcloud = "^0.18.3"
 pathlib = "^1.0.1"
+pyarrow = "^17.0.0"
+pandas = "^2.2.2"
+h5py = "^3.11.0"
 
 [tool.poetry.group.dev.dependencies]
```

Comment:
You can add a [tool.poetry.scripts] section here so that when the package is installed it's also installed in the command line

https://python-poetry.org/docs/pyproject/#scripts

Reply:
I'll work on that in a separate PR for packaging.